backend/btrixcloud/users.py

```diff
@@ -476,6 +478,17 @@ async def change_email_name(
 
         await self.update_email_name(user, user_update.email, user_update.name)
 
+        # If name changed, update any references in crawls and workflows
+        if user_update.name:
```

Comment:
Nice! This definitely works, but seems like for consistency we should move these into the respective BaseCrawlOps and CrawlConfigOps functions?
It's a bit more work, but maybe will be more maintainable.

Was surprised by how fast this was on dev, probably don't need any further optimizations, otherwise could use `asyncio.gather` if it ever become an issue.

Reply:
> Nice! This definitely works, but seems like for consistency we should move these into the respective BaseCrawlOps and CrawlConfigOps functions? It's a bit more work, but maybe will be more maintainable.

Yup, fair enough! I'll make that change.

> Was surprised by how fast this was on dev, probably don't need any further optimizations, otherwise could use `asyncio.gather` if it ever become an issue.

Yeah it's just a few `update_many` calls and dbs are fast at that sort of thing so should always be quick!